library.properties

```diff
@@ -2,10 +2,11 @@ name=PhysicsLabFirmware
 version=1.0.0
 author=Arduino
 maintainer=Arduino <info@arduino.cc>
-sentence=Firmware for the Physics Lab kit.
-paragraph=This library depends on the ArduinoBLE and Adafruit LSM9DS1 Library.
+sentence=Firmware for the [Arduino Science Kit Physics Lab](https://www.arduino.cc/education/science-kit).
+paragraph=This library depends on the ArduinoBLE and MKRIMU libraries, and can be found through the Arduino Science Journal library. We recommend to use it with the following Arduino boards:
```

Comment:
```suggestion
paragraph=This library depends on the ArduinoBLE and MKRIMU libraries, and can be found through the Arduino Science Journal library. We recommend to use it with the following Arduino boards: the Arduino MKR WiFi 1010 + Science Carrier Board included in the Arduino Science Kit or the Arduino Nano 33 BLE.
```

Reply:
This line has the same issue as the preceding one of true spaces having been replaced by non-breaking spaces during your edit.

The claim that this library depends on the MKRIMU library is incorrect. It was changed to using the "Adafruit LSM9DS1 Library" back in 2019:
https://github.com/arduino-libraries/PhysicsLabFirmware/commit/fd0ae6cfdee4f5efbee0015d51f08bb4cc5d9635
So the previous text about dependencies was correct.

You could even remove that text altogether if you like because Arduino Library Manager now automatically installs the library dependencies so the user doesn't really need to be concerned with the library's dependencies.